api/r/cellxgene.census/DESCRIPTION

```diff
@@ -14,12 +14,14 @@ Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.2.3
 Additional_repositories: https://tiledb-inc.r-universe.dev
+biocViews:
```

Comment:
(non-blocking) what does this do?

Reply:
allows installing SingleCellExperiment from bioconductor